pkg/netlink/netlink.go

```diff
@@ -59,7 +58,7 @@ type Basic interface {
 type Interface interface {
 	Basic
 	AddrAddIfNotPresent(link netlink.Link, addr *netlink.Addr) error
-	RuleAddIfNotPresent(rule *netlink.Rule) error
+	RuleAddIfNotPresent(tableID int) error
```

Comment:
As outlined above, keep this as is. 

Also we can add a corresponding delete function: `RuleDelIfPresent`.